packages/react-auth/src/shared/utils.ts

```diff
@@ -55,3 +55,22 @@ export const toUrlEncoded = (obj: { [key: string]: string }): string => {
     )
     .join("&");
 };
+
+/**
+ * isLocalhost - helper function to determine if a hostname is localhost
+ * @param hostname {string}
+ * @returns {boolean}
+ * @example
+ * isLocalhost("pangea.cloud");
+ * // false
+ *
+ * isLocalhost("127.0.0.1");
+ * // true
+ */
+export const isLocalhost = (hostname: string): boolean => {
+  if (hostname === "localhost" || hostname === "127.0.0.1") {
```

Comment:
I know I was the one that wrote this, but wondering if it's worth adding a check for `0.0.0.0` here too?

Reply:
i think `localhost` and `127.0.0.1` is good